tests/test_error_correction.py

```
@@ -0,0 +1,515 @@
+from typing import TYPE_CHECKING
+
+import mqt.qcec
+import pytest
+from mqt.qcec.pyqcec import EquivalenceCriterion
+from qiskit import QuantumCircuit
+from qiskit.circuit import CircuitInstruction, ClassicalRegister
+from qiskit.circuit.library import CXGate, CZGate, HGate, SGate, XGate, ZGate
+from qiskit.quantum_info import hellinger_fidelity
+from qiskit_aer.primitives import SamplerV2
+
+import mqt.bench.benchmark_generation as benchmark_generation
```

Ruff (0.15.15)

[warning] Use `from mqt.bench import benchmark_generation` in lieu of alias

Replace with `from mqt.bench import benchmark_generation`

(PLR0402)